server/src/main/java/org/elasticsearch/action/admin/indices/create/CreateIndexRequest.java

```diff
@@ -109,6 +111,11 @@ public CreateIndexRequest(StreamInput in) throws IOException {
         } else {
             requireDataStream = false;
         }
+        if (in.getTransportVersion().onOrAfter(TransportVersions.FAILURE_STORE_LAZY_CREATION)) {
+            initializeFailureStore = in.readBoolean();
+        } else {
+            initializeFailureStore = false;
```

Comment:
I believe this should be `true`. This is a bit of a nit, since this is behind a feature flag, but technically, a node before this version would have the intent to initialise a failure store by default. Right?